src/main/java/com/pnudev/springonlinedelivery/service/MenuItemServiceImpl.java

```diff
@@ -26,18 +29,23 @@ public List<MenuItemDto> getMenuItems() {
     }
 
     @Override
-    public void postMenuItem(MenuItem menuItem) {
-        // TODO TBW
+    public MenuItemDto postMenuItem(MenuItemDto menuItemDto) {
+        MenuItem menuItem = menuItemMapper.dtoToEntity(menuItemDto);
+        MenuItem savedMenuItem = menuItemRepository.save(menuItem);
+        return menuItemMapper.entityToDto(savedMenuItem);
     }
 
     @Override
-    public void putMenuItem(MenuItem menuItem) {
-        // TODO TBW
+    public MenuItemDto putMenuItem(Long id, MenuItemUpdateDto menuItemUpdateDto) {
+        MenuItem menuItem = menuItemRepository.findById(id)
+                .orElseThrow(() -> new EntityNotFoundException(String.format(MENU_ITEM_MESSAGE, id)));
```

Comment:
instead of `String.format()` use `formatted()`, it has the same syntax but looks nicer